plugins/roles_review.py

```diff
@@ -377,21 +377,30 @@ async def on_member_update(self, before: Member, after: Member) -> None:
     @cleanup
     @command("review_queue")
     @privileged
-    async def review_queue(self, ctx: Context) -> None:
+    async def review_queue(self, ctx: Context, whose: Literal["any", "mine"] = "mine") -> None:
         """List unresolved applications."""
         async with sessionmaker() as session:
-            stmt = select(Application).where(Application.decision == None).order_by(Application.listing_id)
+            if whose == "any":
+                stmt = select(Application).where(Application.decision == None).order_by(Application.listing_id)
+            else:
+                stmt = select(Application).join(Vote, isouter=True) \
+                    .where(Application.decision == None) \
+                    .where((Vote.voter_id != ctx.author.id) | (Vote.voter_id == None)) \
+                    .distinct().order_by(Application.listing_id)
```

Comment:
Took me a minute to process what this does. I feel like a cleaner solution would be to
```py
.outerjoin(Vote, Vote.application_id.and_(Vote.voter_id == ctx.author.id))
```
whence we would get a `NULL` iff there is no vote for the current user. This also removes the need for `.distinct()`.

Reply:
You still have to check that the outerjoin failed (that we've got a NULL)

Reply:
fixed